integrations/docker/images/stage-2/chip-build-nuttx/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG VERSION=1
+FROM ghcr.io/project-chip/chip-build:${VERSION}
+LABEL org.opencontainers.image.source https://github.com/project-chip/connectedhomeip
+
+RUN set -x \
+    && mv /usr/lib/python3/dist-packages/apt_pkg.cpython-38-x86_64-linux-gnu.so /usr/lib/python3/dist-packages/apt_pkg.cpython-39-x86_64-linux-gnu.so \
```

Comment:
this mv looks suspect/odd. Why is this done?

Reply:
The python3 of ghcr.io/project-chip/chip-build docker image is  version 3.9, the add-apt-repository command cannot be executed without rename the library. The message "import apt_pkg not found" is displayed.

Reply:
We do install packages for other bits tough. Is this because we do not use apt? This feels as a workaround rather than a complete fix.

Reply:
If we do not invoke the add-apt-repository command, you do not need to fix this issue.